src/api/todos.ts

```diff
@@ -0,0 +1,22 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = ({ title, completed, userId }: Omit<Todo, 'id'>) => {
+  return client.post<Todo>('/todos', { title, completed, userId });
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const updatedTodo = ({
+  id, title, completed, userId,
+}: Todo) => {
+  const trimmedTitle = title.trim();
```

Comment:
it's better to prepare all data before. This function shouldn't manipulate data, just send request (single responsibility principle)

Reply:
not fixed
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/86923089/501eabac-1da8-491d-bb2f-acdcc211cbf2)